### ISSUE TITLE
Quill-image-resize-module resize works but doesnt update data binding

### ISSUE THREAD

**Issue by jagabs** [KillerCodeMonkey/ngx-quill#145]
Hi i know this isn't part of ngx-quill, but maybe you have a solution.

Before alignment:
![screen shot 2018-03-26 at 10 56 51 am](https://user-images.githubusercontent.com/17292042/37884834-74078d8e-30e4-11e8-8729-c878c00bba5b.png)

After alignment in image-resize-module:
![screen shot 2018-03-26 at 10 57 38 am](https://user-images.githubusercontent.com/17292042/37884847-8cd412ba-30e4-11e8-9adb-e5593c7a0844.png)

it doesnt update the data, so when i save it its still the same

**Comment by KillerCodeMonkey** [comment]
maybe override the imageHandler? or listen on onSelectionChanged and if new range is null you are not longer focusing the editor an grab latest values before saving.

Maybe you can provide an example on plunkr or somewhere else?

**Comment by jagabs** [comment]
I dont know how to set up stackblitz/punker im so sorry. Anyway ive edited your code in the example and it looks like this, 

---HTML---
<h3>Reactive Forms and patch value</h3>
<button type="button" (click)="hide=!(!!hide)">hide / show</button>
<form [formGroup]="form">
  {{form.controls.editor.value}}
  <button type="button" (click)="patchValue()">patchValue</button>
  <button type="button" (click)="setControl()">setControl</button>

  <quill-editor #editor 
      [style.display]="hide ? 'none' : 'block'" 
      [formControl]="form.controls['editor']"
      [modules]="modules"
      ></quill-editor>
</form>

this.modules = {
      toolbar: [
        ['bold', 'italic', 'underline', 'strike'],        // toggled buttons
        ['blockquote', 'code-block'],

        [{ 'header': 1 }, { 'header': 2 }],               // custom button values
        [{ 'list': 'ordered' }, { 'list': 'bullet' }],
        [{ 'script': 'sub' }, { 'script': 'super' }],      // superscript/subscript
        [{ 'indent': '-1' }, { 'indent': '+1' }],          // outdent/indent
        [{ 'direction': 'rtl' }],                         // text direction

        [{ 'size': ['small', false, 'large', 'huge'] }],  // custom dropdown
        [{ 'header': [1, 2, 3, 4, 5, 6, false] }],

        [{ 'color': [] }, { 'background': [] }],          // dropdown with defaults from theme
        [{ 'font': [] }],
        [{ 'align': [] }],

        ['clean'],                                         // remove formatting button

        ['link', 'image', 'video']                         // link and image, video
      ],
      imageResize: {},
}


basically i added the imageResize on the reactive forms and see if the form "form.controls['editor']" updates when the image moves. Thank you very much i really appreciate this since im new to github

**Comment by KillerCodeMonkey** [comment]
could you test this with plain quilljs? without angular - a simple html page on check if quills contentChanged event ist triggered?

i simply listen to ```this.quillEditor.on('text-change', callback)``` and when the resize module does not trigger the quilljs text-change event i am not able to update the model/control ;)

**Comment by jagabs** [comment]
sure sure thanks bro

**Comment by Faiyajz** [comment]
Quill-image-resize-module resize works and update data binding except style.
resize the image is ok but it doesn't get the style like I wanted to set the image in the center.
What should I do ?
